src/clj/puppetlabs/services/jruby/jruby_metrics_core.clj

```diff
@@ -163,7 +163,10 @@
   If the JRuby instance was borrowed to service a HTTP request, then
   the timer name is generated from the sanitized Comidi :route-id.
 
-  Returns a timer named 'other' if the borrow reason does not match
+  If a Clojure Keyword was used as the JRuby borrow reason, then the
+  timer is generated from the Keyword's name and namespace.
+
+  Returns a named labeled 'other' if the borrow reason does not match
```

Comment:
Looks like we lost the word "timer" here.

Reply:
Fixed!